source/standards/storing-source-code/index.html.md.erb

```diff
@@ -6,33 +6,297 @@ old_paths:
+
+How this source code is stored and structured will vary by project, based on needs and historical convention.
+For example:
+
+- A project may be composed of multiple services where each has its own repository
+- A monolith may have all source code for all purposes stored within the same source code repository
+- A mixture may apply where project source code and tests are within one repository,
+  while infrastructure code may be stored within a separate repository
+
+## Source Code Versioning: Git
+
+At the Department for Education (DfE) we use [Git](https://git-scm.com/) for source code versioning.
+
+- Git is decentralised - this means all copies of the repository include the **whole** history of the repository,
+  not just a snapshot
+- Branches are "cheap" - creating a new branch (or tag) involves just a new pointer at a specific commit
+  (thus minimal compute and storage implications)
+- Hashes/checkums for each file and commit depend on the entire tree - thus, the repository is safe from
+  surreptitious / malicious / accidental changes to earlier versions of a file without it being
+  very visible to other users
+
+## Git Repository Hosting - GitHub and Azure DevOps (ADO)
+
+While not required, most git users will nominate one copy of the git repository to be the authoritative copy.
+
+- It is possible to self-host a git server for this purpose but, often, this will be a hosted solution such as
+  GitHub, Azure DevOps, GitLab, or any of the numerous other commercial services available.
+- A "hub and spoke" is easier to reason about and keep synchronised
+- Integrations with other tools will work with less friction, where they have a single copy to work with
+  (e.g., automated test/deployment tools, issue/bug management)
+
+### GitHub
+
+Historically, some projects use (and may remain on) private Azure DevOps and/or private GitHub repositories
+for legacy reasons, though we are now required
+to [make new source code open](https://apply-the-service-standard.education.gov.uk/service-standard/12-make-new-source-code-open.html).
+
+Specifically, we use GitHub for new and migrated work.
+
+### GitHub Organisations
+
+Department for Education (DfE) source code repositories on GitHub should be stored under an appropriate
+organisation, thereby giving appropriate oversight and protections to these source code repositories.
+
+Specifically:
+
+- The [Department for Education Digital](https://github.com/DfE-Digital)
+  GitHub organisation is used for new and existing source code repositories
+  - This is applicable to production and prototype code
+- Work created outside the DfE Digital organisation should be transferred into
+  the DfE Digital organisation at the earliest opportunity.
+  - [GitHub: Guide to transferring a repository](https://help.github.com/en/articles/transferring-a-repository)
+- The [Skills Funding Agency](https://github.com/SkillsFundingAgency/)
+  GitHub organisation is also used by the DfE
```

Comment:
There's also @DFEAGILEDEVOPS - still seems to be used _somewhere_.

Reply:
Added